flink-cdc-common/src/main/java/org/apache/flink/cdc/common/event/SchemaChangeEventType.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.cdc.common.event;
+
+/** An enumeration of schema change event types for {@link SchemaChangeEvent}. */
+public enum SchemaChangeEventType {
+    ADD_COLUMN,
+    ALTER_COLUMN_TYPE,
+    CREATE_TABLE,
+    DROP_COLUMN,
+    RENAME_COLUMN;
+
+    public static SchemaChangeEventType ofEvent(SchemaChangeEvent event) {
+        if (event instanceof AddColumnEvent) {
+            return ADD_COLUMN;
+        } else if (event instanceof AlterColumnTypeEvent) {
+            return ALTER_COLUMN_TYPE;
+        } else if (event instanceof CreateTableEvent) {
+            return CREATE_TABLE;
+        } else if (event instanceof DropColumnEvent) {
+            return DROP_COLUMN;
+        } else if (event instanceof RenameColumnEvent) {
+            return RENAME_COLUMN;
+        } else {
+            throw new RuntimeException("Unknown schema change event type: " + event.getClass());
```

Comment:
Why a RuntimeException thrown here but `ofTag()` method return a `null` object?

Reply:
I thought since `SchemaChangeEventType#ofEvent` receives a `SchemaChangeEvent` which must correspond to a valid event type, any unknown event type is unexpected and implies some coding mistake. However, `SchemaChangeEventType#ofTag` receives a String which might not be a valid type, so I didn't throw an exception for that case. Maybe `Optional<>` is better?

Reply:
We can simply throw a RuntimeException for invalid String tag, a null SchemaChangeEventType is dangerous for caller.